lib/config/types.ts

```diff
@@ -357,6 +357,16 @@ export interface RenovateOptionBase {
   experimentalDescription?: string;
 
   experimentalIssues?: number[];
+
+  advancedUse?: boolean;
+
+  followTagAdvancedFunctionality?: boolean;
+
+  groupAdvancedFunctionality?: boolean;
+
+  advancedConfig?: boolean;
+
+  commitMsg?: boolean;
```

Comment:
Do we usually abbreviate the word `message` to `msg`? I would expect it to be written out fully like this: `commitMessage`.

Also I don't really understand what this Boolean is doing, just by looking at the name `commitMsg`. It looks like we want to nudge our users away from editing the commit message themselves?

How about calling it: `directEditOfCommitMessageDeprecation`? Or at least a more descriptive name. 😉